proto/wg/cosmo/graphqlmetrics/v1/graphqlmetrics.proto

```diff
@@ -0,0 +1,48 @@
+// https://protobuf.dev/programming-guides/style/
+
+syntax = "proto3";
+
+package wg.cosmo.graphqlmetrics.v1;
+
+import "wg/cosmo/common/common.proto";
+
+message SchemaUsageInfo {
+  string OperationDocument = 1;
```

Comment:
I would name this RequestDocument.
(A GraphQL Document can contain Operations as per the GraphQL spec)

Reply:
Done